package_testing/test.py

```diff
@@ -0,0 +1,26 @@
+from core.feluda import Feluda
+from sklearn.metrics.pairwise import cosine_similarity
+from  core.models.media_factory import ImageFactory
+
+feluda = Feluda('config.yaml')
+feluda.setup()
+
+feluda.setup()
```

Comment:
there is a repetition of `feluda.setup()`

Reply:
I'll make changes right away